lib/rules/selector-max-specificity/README.md

```diff
@@ -131,17 +131,17 @@ The following patterns are considered problems:
 
 <!-- prettier-ignore -->
 ```css
-:global(.foo) .bar.baz {}
+:host(.foo) .bar.baz {}
 ```
 
 <!-- prettier-ignore -->
 ```css
-:local(.foo.bar.baz) {}
+:host-context(.foo.bar.baz) {}
 ```
 
 <!-- prettier-ignore -->
 ```css
-:local(.foo, :global(.bar), .foo.bar.baz) {}
+:host-context(.foo, :host(.bar), .foo.bar.baz) {}
```

Comment:
nit

```suggestion
:host-context(.foo, :host(.bar), .baz.qux) {}
```

Reply:
This has one class selector less and wouldn't produce an error.